Foodpanda-463/styles.scss

```diff
@@ -0,0 +1,958 @@
+        align-items: center;
+        position: relative;
+
+        .title {
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+            height: 40%;
+            width: 55%;
+            position: absolute;
+            top: 0;
+
+            .dash,
+            .for,
+            .gold,
+            .sparkle,
+            .cross {
+                position: absolute;
+                width: 100%;
+                top: 0;
+                opacity: 0;
+            }
+
+            .dash {
+                transform: translateY(9%);
+            }
+
+
+            .sparkleItem1,
+            .sparkleItem2,
+            .sparkleItem3,
+            .sparkleItem4,
+            .sparkleItem5 {
+                opacity: 0;
+                transform: scale(0.2);
+                animation: titleUp 1.5s linear 1s forwards, sparkleTitle 5s linear 1s infinite;
+            }
+
+            .sparkleItem6 {
+                opacity: 0;
+                transform: scale(0.2);
+                animation: titleUp 1.5s linear 1s forwards, sparkleTitle 5s linear infinite;
+            }
+
+            @keyframes titleUp {
+                40%,
+                100% {
+                    opacity: 1;
+                    transform: translateY(0) scale(1);
+                }
+            }
+
+            @keyframes sparkleTitle {
+                0%,
+                30%,
+                70%,
+                100% {
+                    opacity: 1;
+                }
+
+                10%,
+                50%,
+                85% {
+                    opacity: 0.7;
+                }
+            }
+
+
+            .for {
+                transform: translateY(13%) scale(0.2);
+                opacity: 0;
+            }
+
+            .gold {
+                transform: translateY(17%) scale(0.2);
+                opacity: 0;
+            }
+
+            .sparkleAnim1,
+            .sparkleAnim2,
+            .sparkleAnim3,
+            .sparkleAnim4,
+            .sparkleAnim5 {
+                opacity: 0;
+            }
+
+            .sparkleAnim1 {
+                animation: sparkleAnim 4s linear 1s infinite;
+            }
+
+            .sparkleAnim2 {
+                animation: sparkleAnim 4s linear 3s infinite;
+            }
+
+            .sparkleAnim3 {
+                animation: sparkleAnim 4s linear 2s infinite;
+            }
+
+            .sparkleAnim4 {
+                animation: sparkleAnim 4s linear 5s infinite;
+            }
+
+            .sparkleAnim5 {
+                animation: sparkleAnim 4s linear 4s infinite;
+            }
+
+            @keyframes sparkleAnim {
+
+                0%,
+                30%,
+                70% {
+                    opacity: 0;
+                }
+
+                20%,
+                40%,
+                80%,
+                100% {
+                    opacity: 1;
+                }
```

Comment:
to prevent the flickering effect, you need the start and end of the infinite animation to be similar. I meen 0% and 100% opacity: 0 or opacity:1